openlibrary/templates/site/head.html

```diff
@@ -4,8 +4,6 @@
 <html xmlns="http://www.w3.org/1999/xhtml" lang="en">
 <head>
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-    <meta name="title" content="" />
-    <meta name="description" content="Open Library is an open, editable library catalog, building towards a web page for every book ever published.">
```

Comment:
Are we sure we want to remove this description? This is different than the `og:description`. If anything, I'd expect us to _add_ a `content` value to the `title` tag rather than remove it.

Reply:
@mekarpeles It is the same description :)

Reply:
`og` is for facebook's open graph protocol. https://moz.com/learn/seo/meta-description `<meta description ...` is the canonical way to specify a page's description for search engines. We want to add these two fields back and probably add a reasonable title. I'l make a suggestion in another comment.